pkg/process/checks/process_common_test.go

```diff
@@ -258,6 +345,69 @@ func TestFormatNetworks(t *testing.T) {
 	}
 }
 
+func TestFormatCPU(t *testing.T) {
+	hostCPUCount = func() int {
+		return 4
+	}
```

Comment:
[minor] Save and reset the previous func that you are overriding incase other tests rely on this.

Reply:
Done!